src/main/java/sirius/biz/jobs/batch/file/LineBasedExportJobFactory.java

```diff
@@ -37,8 +37,4 @@ public String getIcon() {
         return "far fa-file-excel";
     }
 
-    @Override
-    public String getLegacyIcon() {
-        return "fa-file-excel-o";
-    }
```

Comment:
Please double check. This was introduced only recently with [OX-9090](https://scireum.myjetbrains.com/youtrack/issue/OX-9090/). If _all_ usages of job icons in wondergem templates should be replaced with `fa-gears`, please remove the `getLegacyIcon()` alltogether.